src/run.f90

```diff
@@ -7761,12 +7761,14 @@ subroutine FileManagement()
 
     integer(int32) :: RepeatToDay
     real(dp) :: WPi
+    logical :: HarvestNow
 
     WPi = 0._dp
+    HarvestNow = .false.
     RepeatToDay = GetSimulation_ToDayNr()
 
     loop: do
-        call AdvanceOneTimeStep(WPi)
+        call AdvanceOneTimeStep(WPi, HarvestNow)
```

Comment:
If an argument is added here, then it also needs to be added into the subroutine AdvanceOneTimeStep, and I do not see that change implemented.

Reply:
(also update description, b/c it says DEBUG=0 twice)

Reply:
HarvestNow was already added to AdvanceOneTimeStep when making the PR (also, this code was compiled and tested). I also updated the description before your review. Maybe you were not seeing the last version of the PR? I remember this sometimes happened during the hackathon